src/BenchmarksApps/Antiforgery/Program.cs

```diff
@@ -0,0 +1,11 @@
+var builder = WebApplication.CreateBuilder(args);
+builder.Logging.ClearProviders();
+var app = builder.Build();
+
+app.UseAntiforgery();
+
+await app.StartAsync();
+
+Console.WriteLine("Application started.");
```

Comment:
You'll need an endpoint that uses antiforgery. https://learn.microsoft.com/en-us/aspnet/core/security/anti-request-forgery?view=aspnetcore-9.0#antiforgery-with-minimal-apis

As well as a way to get an antiforgery token for the client to use in its requests.

Reply:
WRK has ways to fetch a token on the first request and the reuse it for subsequent requests. Check the LUA scripts we have in this repos for auth.

Another option could be to use K6 but I am afraid its perf to be a bottleneck.